Factory/Config/StackFactoryInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Factory\Config;
+
+use Liip\ImagineBundle\Config\FilterInterface;
+use Liip\ImagineBundle\Config\Stack;
+
+interface StackFactoryInterface
+{
+    /**
+     * @param string            $name
+     * @param string|null       $dataLoader
+     * @param int|null          $quality
+     * @param FilterInterface[] $filters
+     *
+     * @return Stack
```

Comment:
could we instead do the return type declaration on the method itself?

Reply:
done